.github/workflows/gha_generate_readme_and_issuetemp.yml

```diff
@@ -19,28 +19,27 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v3
 
-      - name: Setup Conda
-        uses: s-weigand/setup-conda@v1
+      - name: Setup Micromamba ${{ matrix.python-version }}
+        uses: mamba-org/setup-micromamba@v1
         with:
-          activate-conda: false
-          conda-channels: conda-forge
-
-      - name: Build environment
-        shell: bash -l {0}
-        run: |
-          conda env create -f environment.yml
+          environment-file: environment.yml
+          init-shell: bash
 
       - name: Generate the README and ISSUE TEMP from JSON source
-        run: |
-          source activate cefi-info-hub
+        shell: bash -l {0}
+        run: >
```

Comment:
I guess that we don't want to execute the next step if the first fails, right? If not I can revert this here and in the `Commit and push if it changed` below.